packages/cli-clean/src/clean.ts

```diff
@@ -1,5 +1,6 @@
 import {getLoader} from '@react-native-community/cli-tools';
 import type {Config as CLIConfig} from '@react-native-community/cli-types';
+import chalk from 'chalk';
```

Comment:
we'll need to add `chalk` to package.json

Reply:
Whoops 😄